File: Data Structures & Algorithms/non-cyclical-number/submission-0.py
class Solution:
    def isHappy(self, n: int) -> bool:
        seen = set()

        total = n
        seen.add(total)
        count = 0
        while(True):
            new = 0
            print(seen)
            print(total)
            for i in list(str(total)):
                new += int(i) * int(i)
            if new == 1:
                return True
            elif new in seen:
                return False
            else:
                seen.add(new)
                total = new


        
        

File: Data Structures & Algorithms/two-integer-sum/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        unordered_map<int, int> diffs;
        int count = 0;
        for (auto i: nums){
            int diff = target - i;
            if(diffs.find(diff) != diffs.end()){
                return {diffs[diff], count};
            }
            diffs.insert({i,count});
            count++;
        }
        return {};
    }
};
